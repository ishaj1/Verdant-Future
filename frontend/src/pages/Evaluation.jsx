import Header from "../components/Header";
import CompanyEvaluationForm from "../components/CompanyEvaluationForm";

export default function EvaluationPage() {
  return (
    <>
      <Header />
      <h1>Request an Evaluation</h1>
      <CompanyEvaluationForm />
    </>
  );
}
