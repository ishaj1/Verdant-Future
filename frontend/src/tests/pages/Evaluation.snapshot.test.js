import React from "react";
import renderer from "react-test-renderer";
import Evaluation from "../../pages/Evaluation";

jest.mock("../../components/Header", () => () => <div data-testid="mock-header">Mock Header</div>);
jest.mock("../../components/CompanyEvaluationForm", () => () => <div data-testid="mock-company-evaluation-form">Mock Company Evaluation Form</div>);

test("Evaluation page matches snapshot", () => {
  const tree = renderer.create(<Evaluation />).toJSON();
  expect(tree).toMatchSnapshot();
});
