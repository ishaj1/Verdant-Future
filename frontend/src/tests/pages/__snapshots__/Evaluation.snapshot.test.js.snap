// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Evaluation page matches snapshot 1`] = `
[
  <div
    data-testid="mock-header"
  >
    Mock Header
  </div>,
  <h1>
    Request an Evaluation
  </h1>,
  <div
    data-testid="mock-company-evaluation-form"
  >
    Mock Company Evaluation Form
  </div>,
]
`;
